charts/kube-state-metrics/templates/deployment.yaml

```
@@ -176,6 +176,9 @@ spec:
         image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
         ports:
         - containerPort: 8080
+{{ if .Values.selfMonitor.enabled -}}
+        - containerPort: 8081
+{{ end -}}
```

Re https://github.com/prometheus-community/helm-charts/pull/903/checks?check_run_id=2452191125

>Error:  templates/deployment.yaml: unable to parse YAML: error converting YAML to JSON: yaml: line 129: did not find expected key

At a quick glance, the trailing dash may be pulling the YAML up to the next line, making the deploy fail?

```suggestion
{{- if .Values.selfMonitor.enabled }}
        - containerPort: 8081
{{- end }}
```

---

Yep, I just caught that rendering the templates locally, and made this change.